src/Bridge.js

```diff
@@ -21,9 +21,9 @@ import GlobalErrorBoundary from 'components/GlobalErrorBoundary';
 
 const INITIAL_NETWORK_TYPE = isRopsten
   ? NETWORK_TYPES.ROPSTEN
-  : isDevelopment
-  ? NETWORK_TYPES.LOCAL
-  : NETWORK_TYPES.MAINNET;
+  : isMainnet
+  ? NETWORK_TYPES.MAINNET
+  : NETWORK_TYPES.LOCAL;
```

Comment:
I don't think we want to disregard the `isDevelopment` flag here, right? Default behavior for production builds should still be mainnet.

```suggestion
  : (isDevelopment && !isMainnet)
  ? NETWORK_TYPES.LOCAL
  : NETWORK_TYPES.MAINNET;
```

Reply:
Good catch, LGTM